hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -275,6 +275,18 @@ public boolean shouldCacheBlockOnRead(BlockCategory category) {
       || (prefetchOnOpen && (category != BlockCategory.META && category != BlockCategory.UNKNOWN));
   }
 
+  public boolean shouldCacheFileBlock(HFileInfo hFileInfo, Configuration conf) {
+    Optional<Boolean> cacheFileBlock = Optional.of(true);
+    // Additionally perform the time-based priority checks to see
+    // whether, or not to cache the block.
+    if (getBlockCache().isPresent()) {
+
+      cacheFileBlock = getBlockCache().get().shouldCacheFile(hFileInfo, conf);
+      LOG.info("BlockCache Present, cacheFileBlock: {}", cacheFileBlock.get());
```

Comment:
Nit: Can we change the log level to debug? This information might only be needed during debugging, and since it is logged for each block, it could generate a lot of logs.

Additionally, is the log information sufficient? It only shows true or false but doesn't indicate which file or block this information corresponds to. It will be a bunch of lines with just 'BlockCache Present, cacheFileBlock: true/false,' not revealing any detailed information.

Also, I am not sure this log line is needed because `shouldCacheFile(hFileInfo, conf)` already logs this information.

Reply:
I had added to to debug something. Needs to be cleaned up. thanks for pointing this out.